template.tpl

```diff
@@ -33,18 +33,15 @@ ___TEMPLATE_PARAMETERS___
     "type": "TEXT",
     "name": "site_name",
     "simpleValueType": true,
-    "help": "If your site is xyz.com please enter xyz in this field.",
```

Comment:
Should we have a help message for this field like we did previously?

Reply:
@raymondborkowski, not sure how this one got removed or how to add it in the UI but i will add it to the template after export.

![image](https://user-images.githubusercontent.com/96016324/159266025-37e3bfc7-bc34-4d28-b9ce-6b9f6342fe5a.png)